psi4/src/psi4/cc/amps.cc

```diff
@@ -32,35 +32,38 @@
 #include "psi4/libmints/dimension.h"
 #include "psi4/libmints/matrix.h"
 #include "psi4/libpsi4util/process.h"
+#include "psi4/libmints/molecule.h"
 #include <memory>
-namespace psi {
-namespace ccenergy {
+namespace psi::ccenergy {
```

Comment:
This is technically a C++17 feature.  We only compile Psi4 with C++14 and strict compilers may complain or error.  Please revert to `namespace psi { namespace ccenergy {`

For others: Are we doing this elsewhere in the code?